assets/js/blocks/product-search/style.scss

```diff
@@ -15,11 +15,11 @@
 		border: none;
 		cursor: pointer;
 		background: none;
-		padding: 8px;
+		padding: 0 0.5em;
```

Comment:
Like with in the _Product Categories List_ block, this button takes all the available vertical space so there is no need to have top and bottom padding. This way both buttons share the same styles (see #2517).